comms/.dockerignore

```diff
@@ -4,7 +4,7 @@ comms-linux
 wip
 .vscode
 *.md
-.git
```

Comment:
why

Reply:
was trying to get circleci build to have the VCS info (sha, build date, etc)...
but the real problem is that we do `COPY . .` and the `.git` folder is actually in the parent dir.
I think there are some options, but this change doesn't really change anything.